src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -45,11 +46,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String lastName = request.getParameter("last-name");
     String username = request.getParameter("username");
     String email = userService.getCurrentUser().getEmail();
-    int major = Integer.parseInt(request.getParameter("major"));
-    Boolean is_mentor = false;
+    int major = Utility.tryParseInt(request.getParameter("major"));
+        // Should not be possible for parseInt() to fail because form only allows integer values to
+        // be submitted.
```

Comment:
Include this comment before the function call (the line above) and make sure indentation is consistent with the rest of this block.

Reply:
Same in your other call site!

Reply:
Done!